src/components/organisms/modals/DeleteRoleModal/DeleteRoleModal.tsx

```diff
@@ -0,0 +1,66 @@
+import { FC, useCallback } from 'react'
+import { AppearanceTypes, SizeTypes } from 'components/molecules/Button/Button'
+import ModalBase, {
+  ButtonPositionTypes,
+  TitleFontTypes,
+} from 'components/organisms/ModalBase/ModalBase'
+
+export interface DeleteRoleModalProps {
+  title?: string
+  cancelButtonContent?: string
+  proceedButtonContent?: string
+  modalContent?: string
+  isModalOpen?: boolean
+  closeModal: () => void
+  onClose?: () => void
```

Comment:
You don't pass the onClose func. What is the purpose of this function?

Reply:
Removed redundant code